vhdbuilder/packer/components.json

```diff
@@ -240,14 +240,14 @@
         "v1.1.1"
       ],
       "multiArchVersions": [
-        "v0.7.11",
-        "v0.7.12",
-        "v1.0.8",
-        "v1.0.9",
-        "v1.1.4",
-        "v1.1.5",
-        "v1.23.1",
-        "v1.23.2"
```

Comment:
We should keep the last updated versions.
```
v0.7.12  // last updated 
v0.7.14  // new one
```

Reply:
Is it correct? Even this time I bump up 2 versions, I still need to keep the last updated version?

Reply:
I think the reason we keep the last updated version is that RP would use it according to its config. We need to update RP and bring it to the latest version. (but still, it could be a gap that RP uses the newest VHD and requires the last updated version)

For this cause, should we bump the latest versions which are `v0.7.14` / `v1.0.11` / `v1.1.7` / `v1.23.4` and the last updated versions? And we don't need other versions such as `v0.7.13`.

@feiskyer would you plz confirm the logic that we keep "two" versions here?

Reply:
yes, we need to keep the last version updated in RP and the latest version.

@lzhecheng please update the versions to today's new releases.

Reply:
Got it.